ConfigFile.xml

```diff
@@ -1,22 +0,0 @@
-  <ConfigFiles>
```

Comment:
This file does not need to be committed as it is an output file from the post service.

Reply:
Sorry, ignor this comment, i see that it is being removed.